Selenium/UserRegister.py

```diff
@@ -0,0 +1,98 @@
+import time
+from selenium import webdriver
+from selenium.webdriver import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
jeżeli selenium bedzie w każdym/prawie każdym pliku to warto to dodać do configu

Reply:
co mi da to, że wrzucę to do configa?

Reply:
no jeżeli config również z domysłu będzie importowany przez każdy plik to można skrócić w ten sposób ilość importów. Ale to tylko propozycja, nie  `reqest changes`